src/Epic/index.tsx

```diff
@@ -72,13 +73,18 @@ export type BrazeMessageProps = {
     remindMeConfirmationHeaderText?: string;
     remindMeConfirmationText?: string;
     hidePaymentIcons?: string;
+    authoredEpicHeader?: string;
+    authoredEpicImageUrl?: string;
+    authoredEpicImageAltText?: string;
+    authoredEpicBylineName?: string;
+    authoredEpicBylineCopy1?: string;
+    authoredEpicBylineCopy2?: string;
```

Comment:
Since these are now props which can be passed through to the regular `Epic` component, I don't think there's now any need for the `EpicWithSpecialHeader` component. I think this is highlighted by the definition of the `EpicWithSpecialHeader` component being simply a call to the `Epic` component:

```js
export const EpicWithSpecialHeader: React.FC<EpicProps> = (props: EpicProps) => <Epic {...props} />;
```

Reply:
For now I'd like to leave it as its own separate component as that's what Marketing expect it to be. There's also the issue where we want some of these new props to be mandatory for the byline epic, but can be ignored for the default epic and I don't know how to solve that in either storybook or in the default epic's `canRender` check